### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by frederoni**
API changed since we last updated the readme.

@ericrwolfe @bsudekum  👀

### CODE REVIEW

**Comment by Svantulden** [comment]
Should the example also import `MapboxCoreNavigation` as the example now uses `SimulatedLocationManager`?

File: README.md [hunk 1]
```diff
@@ -74,9 +75,9 @@ options.includesSteps = true
 
 Directions.shared.calculate(options) { (waypoints, routes, error) in
     guard let route = routes?.first else { return }
-
-    let viewController = NavigationViewController(for: route)
-    viewController.simulatesLocationUpdates = true
+    
+    let locationManager = SimulatedLocationManager(route: route)
```
Comment (bsudekum) [inline README.md]: @frederoni what's your thinking behind adding `SimulatedLocationManager` here? I'm sort of thinking we want to make the example that gets the most eyes, the most barebones example we can come up with.

Comment (frederoni) [inline README.md]: I agree. Let's remove the simulation.

Comment (frederoni) [inline README.md]: fixed

**Pull request merged**